eras/alonzo/impl/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Version history for `cardano-ledger-alonzo`
 
+## 1.10.1.1
+
+* Added `wireSizeAlonzoTxF`
+
 ## 1.10.1.0
 
```

Comment:
`1.10.1.0` has not been released yet. Again, new sections in the changelogs are never added. Also addition of a new function is a minor version bump. Patch version bump is reserved for non-semantic or non visible changes (eg, tests, documentation etc.)

```suggestion
## 1.10.1.0

* Added `wireSizeAlonzoTxF`

```

Reply:
Oops, my bad, I've read releasing guide and looked at how some other commits look like before making my commits and I guess I didn't quite catch it.